### PR TITLE
Update questions

### LINKED ISSUE #114: Refactor questions to include symbols
Some questions are confusing due to inputs only allowing numbers, but answers would include a symbol ($, %). Those symbols need to be added to the inputs so that users know they don't need to add them themselves.

### LINKED ISSUE #116: Revise Principal/Rent questions to be an either/or

### PR DESCRIPTION
**Pull request by timkeresey**
### What does this PR do?

- Adds to styling of Question.
- Makes improvements to the question UI.
- Creates a new page element for either/or question in the question set.

### Where should the reviewer start?

- Question.jsx

### How should this be manually tested?

- Navigating the App in the browser

### What are the relevant tickets?

-  #115 
- closes #114 
- closes #116

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -71,7 +71,12 @@ const App: React.FC = () => {
             <animated.div key={key} style={props}>
               <Switch location={item}>
                 <Redirect exact from="/" to="/home" />
-                <Route exact path="/profile" component={Profile} />
+                <Route
+                  exact path="/profile"
+                  component={() => (
+                    <Profile updateReport={updateReport} />
+                  )}
+                />
```
Comment (Atos20) [inline src/components/App/App.tsx:79]: working on this to only appear once the user has filled out the questionnaire

File: src/components/Question/Question.tsx [hunk 2]
```diff
@@ -13,43 +13,59 @@ interface Props {
...
+      setIndex(7);
+      updateAnswer({ ...answerInput, [questionKeys[8]]: '0'});
+    } else if (e.target.id === 'rent') {
+      setPrincipleSelected(false);
+      setRentSelected(true);
+      setIndex(8);
+      updateAnswer({ ...answerInput, [questionKeys[7]]: '0'});
+    }
+  }
+
```
Comment (Atos20) [inline src/components/Question/Question.tsx:59]: This was a great addition to handle the question's logic. Thanks for your hard work!!

File: src/components/Question/Question.tsx [hunk 3]
```diff
@@ -13,43 +13,59 @@ interface Props {
...
   const questionContext = useContext(QuestionContext);
-  const questionKeys = Object.keys(questionContext )
+  const questionKeys = Object.keys(questionContext)
   const [answerInput, updateAnswer] = useState<any>({});
   const [index, setIndex] = useState<number>(0);
+  const[isSelected, setIsSelected] = useState<boolean>(false)
   const [errorMessage, setErrorMessage] = useState<string>("");
+  const[rentSelected, setRentSelected] = useState<boolean>(false);
+  const[principleSelected, setPrincipleSelected] = useState<boolean>(false)
   const inputRef = useRef<HTMLInputElement>(null);
```
Comment (Atos20) [inline src/components/Question/Question.tsx:23]: I liked the use of the state to know when the user has selected one of the other.

**Review by Atos20 — COMMENTED** [review]
Great Job Tim this looks fantastic!!

**Review by BlakeDonn — APPROVED** [review]
Resolved conflicts, tested that buttons are functional for rent or goal home.

**Pull request merged**